aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -10,7 +10,7 @@
     /* global Cl, describe, it, expect */
 
     // ########################################################################
-    describe('cl.events.js', function () {
+    describe('cl.events.js:', function () {
```

Comment:
just saying feel free to change it, you can also consult with @evildmp to create the structure. In the end it needs to be readable